src/Utility/Template.php

```diff
@@ -22,9 +22,9 @@ public function renderTemplateStringWithContext(string $template, array $context
 
             return $twig->render('template', $context);
         } catch (Exception $e) {
-            error_log($e->getMessage());
+            error_log('CHANGELOG GENERATION FAILURE: ' . $e->getMessage());
```

Comment:
Seems like a weird flow now. Catch the exception, that would exit given it wasn't caught, and then exit. Can't you just remove this entire catch block for the same effect?

Reply:
I suppose I could. I'm in the habit of catching exceptions any time they could be thrown, even if the output I produce is only slightly less angry 😅